### PR TITLE
[WIP] Add spanish nouns verbs animals

### PR DESCRIPTION
**Pull request by 0xlucyfer**
Use this as a placeholders for discussion & list accumulation/reference for the spanish community.

This MR translates existing ens.vision English nouns & verb lists into Spanish lists.
This MR creates an English animals list.
This MR copies English animals list into Spanish.

Google translate isn't 100% accurate. Suspect translations can be checked against https://www.collinsdictionary.com/us/translator.

- [x] Translate existing English Noun into Spanish.
- [x] Translate existing English Verbs into Spanish.
- [x] Create English animals list.
- [x] Translate English Animals into Spanish.

## References
https://github.com/Zimtente/ens-collections/blob/main/collections/english-nouns.csv
https://github.com/Zimtente/ens-collections/blob/main/collections/english-verbs.csv

#### Example of incorrect translations
Screen Shot 2022-07-27 at 5 17 15 PM
Screen Shot 2022-07-27 at 5 00 48 PM

### CODE REVIEW

**Comment by 0xlucyfer** [comment]
Please feel free to reach out to me with any questions or feedback! Cheers

**Comment by CryptoGatsu** [comment]
king!

**Comment by Zimtente** [comment]
def. a good list to add to our collection. We need more "general" lists like that. Good job! looking forward to adding that list!

**Comment by 0xlucyfer** [comment]
Thanks y'all! 

Small changed needed 
in line 1267 of the collections.json change `},` to `}`.

I plan on adding a few more english/spanish lists.

Cheers!

File: collections/english-nouns.csv [hunk 1]
```diff
@@ -2961,3 +2961,4 @@ merchandize,80839904129547128973182661428256906697287917300863854752579751281336
 kidnaper,69957007819168817257000086629248135067829535909481837772152367512202408838157
 microsphere,12430592501553366531366193527297595727176024872440078439288020661489291832438
 pentode,54801904824765800759815167047049974586957899357641518587936460357380582290087
+cartel,b05e621ee2e5e6d6119e0a987087ed32b6c6fe1113b181eaa85ede67faddbdd5
```
Comment (szanuje) [inline collections/english-nouns.csv:2964]: Wait, how these was merged. It is not a token ID. @CryptoGatsu @Zimtente

Comment (0xlucyfer) [reply collections/english-nouns.csv:2964]: What do you mean ser?? If you run the code below it returns the hash provided. 

```
var namehash = require('@ensdomains/eth-ens-namehash');
var hash = namehash.hash('cartel')
hash = hash.replace('0x', '');
console.log('cartel' + "," + hash)
```

https://www.npmjs.com/package/@ensdomains/eth-ens-namehash

Please let me know if there is a mis-understanding on my end

Comment (0xlucyfer) [reply collections/english-nouns.csv:2964]: I was mistaken. The correct code for generating a valid hash is below 

```
const ethers = require('ethers')
const BigNumber = ethers.BigNumber
const utils = ethers.utils
const name = 'cartel'
const labelHash = utils.keccak256(utils.toUtf8Bytes(name))
const tokenId = BigNumber.from(labelHash).toString()
console.log('cartel' + "," + tokenId)
```

I will update this mr with correct hashes later today. Thanks for putting some eyes on this!

This should be the correct entry?

`
cartel,24930356794076149583364840017866382670785817347831594360562413046882721988203
`

Comment (0xlucyfer) [reply collections/english-nouns.csv:2964]: Please see my updated MR

https://github.com/Zimtente/ens-collections/pull/450

Comment (0xlucyfer) [reply collections/english-nouns.csv:2964]: New MR to fix hashes has been merged into main. Will resolve this conversation.

**Pull request merged**